project/README.md

```diff
@@ -1,17 +1,17 @@
 # Hacking on Docker
 
-The hack/ directory holds information and tools for everyone involved in the process of creating and
+The `project/` directory holds information and tools for everyone involved in the process of creating and
 distributing Docker, specifically:
 
 ## Guides
 
-If you're a *contributor* or aspiring contributor, you should read CONTRIBUTORS.md.
+If you're a *contributor* or aspiring contributor, you should read [CONTRIBUTORS.md](./CONTRIBUTORS.md).
```

Comment:
I think this should be `../CONTRIBUTING.md`, because `./CONTRIBUTORS.md` is a symlink